setup.py

```diff
@@ -55,6 +56,7 @@ def read(*rnames):
             'zest.releaser',
         ],
         'test': [
+            'Products.Archetypes',
```

Comment:
Sure; we still depend on it for the demo content. If you have a better idea, please submit the code ;)